src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
+1 to delaying the soft takeover ignore-next support to a future PR (or delaying soft takeover of the metaknob entirely for now). 

I think we should look for a more generic fix like adding soft takeover support to COs and making force/ignore part of the control api if there's a strong need for it. Using multiple controls introduces the possibility of races since the GUI and controller thread could change them concurrently.

We also should feature freeze 2.1 soon and I think some of this would benefit from a longer period of experimentation in master while effect metaknobs seem pretty fine to include in 2.1 without much risk of wanting to go back on it later.